src/components/bookReact.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+const dummyBooks = [
+  {
+    id: 123,
+    title: 'This is a book title',
+    auhor: 'Alex',
+  },
+];
+
+const DisplayBook = () => {
+  const [book, setBook] = useState(dummyBooks);
+  return (
+    <div>
+      <section>
+        <p>{book[0].title}</p>
+        <ul>
```

Comment:
- [ ] Your DisplayBook component is expected to take book data as a prop and render the title and the author. Kindly fix this.
- [ ] Your component should also have a button for removing books. This button does not have to work yet. This is merely a placeholder for now

Reply:
Hi @TedLivist thanks for your feedback, I'm using functional components, instead use props, I'm using hooks, is it fine? 
For the button, taking the picture in the sneak peek as a guide, I'll use the <code>li</code> Remove as a button to delete an element, is it fine? 🚫 

Thanks for your comments 😃